src/accordion/panel.js

```diff
@@ -61,7 +61,14 @@ class Panel extends React.Component<PanelPropsT> {
   }
 
   render() {
-    const {expanded, disabled, overrides = {}, children, title} = this.props;
+    const {
+      expanded,
+      id,
```

Comment:
should we call this `aria-controls`, if used for that? I believe we follow a practice like that in other components too

Reply:
Updated 👍